modules/wildfly/pom.xml

```diff
@@ -95,13 +95,12 @@
             <artifactId>hibernate-envers</artifactId>
             <optional>true</optional>
         </dependency>
-        <!-- This is to satisfy Neo4J's custom Lucene,
-          not to be confused with the one used by Hibernate Search -->
+        <!-- This is to satisfy Neo4J's custom Lucene: might not match the needs of Hibernate Search -->
```

Comment:
Ok, got it. I think the "satisfy" is what confused me. You actually override Neo4j's own version.